### PR TITLE
octobuild.octobuild version 0.4.0

### PR DESCRIPTION
**Pull request by slonopotamus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95039)

### CODE REVIEW

File: manifests/o/octobuild/octobuild/0.4.0/octobuild.octobuild.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: octobuild.octobuild
+PackageVersion: 0.4.0
+PackageLocale: en-US
+Publisher: Marat Radchenko
```
Comment (ImJoakim) [inline manifests/o/octobuild/octobuild/0.4.0/octobuild.octobuild.locale.en-US.yaml:7]:
```suggestion
Publisher: Artem V. Navrotskiy
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
To match ARP
```
Name          : octobuild
Publisher     : Artem V. Navrotskiy
Version       : 0.4.0
```

File: manifests/o/octobuild/octobuild/0.4.0/octobuild.octobuild.locale.en-US.yaml (hunk 1 above)
Comment (slonopotamus) [reply manifests/o/octobuild/octobuild/0.4.0/octobuild.octobuild.locale.en-US.yaml]: I'm not sure what is ARP, but okay.

Comment (ImJoakim) [reply manifests/o/octobuild/octobuild/0.4.0/octobuild.octobuild.locale.en-US.yaml]: > I'm not sure what is ARP, but okay.

My apologies, I should've been more clear on that. ARP stands for Add or Remove Program, the list of installed programs. The easiest way to get to this menu is via Run -> `appwiz.cpl` or open up the control panel and click on "Uninstall a program".

image
(It's in Norwegian, Navn = Name, Utgiver = Publisher).

The regedit string for this this specific application is located here:
`HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{BCA869D2-6FDD-4896-B1A2-C9F47132BED0}`

image

**Review by ImJoakim — APPROVED** [review]

File: manifests/o/octobuild/octobuild/0.4.0/octobuild.octobuild.locale.en-US.yaml (hunk 1 above)
Comment (slonopotamus) [reply manifests/o/octobuild/octobuild/0.4.0/octobuild.octobuild.locale.en-US.yaml]: Oh, okay.

**Pull request merged**